app/models/search/search_counts.rb

```diff
@@ -71,6 +71,20 @@ def work_cache_key(owner)
     "work_count_#{owner.model_name.cache_key}_#{owner.id}_#{logged_in}"
   end
 
+  ######################################################################
+  # COLLECTION COUNTS FOR USER/PSEUD DASHBOARD
+  ######################################################################
+
+  def collection_count_for_user(user)
+    Rails.cache.fetch(["user", user.id, "collections_count"], dashboard_cache_options) do
+      CollectionSearchForm.new(maintainer_id: user.id, sort_column: "title.keyword").search_results.scope(:for_search).total_entries
+    end
+  end
+
+  def collections_cache_key(owner)
+    "collections_count_#{owner.model_name.cache_key}_#{owner.id}_#{logged_in}"
+  end
```

Comment:
It doesn't look like this is being used, which is fine. Unlike with bookmarks or works, we don't need to change the number based on who is looking at the page. This method can be removed.